client/src/components/answer/Answer.tsx

```diff
@@ -0,0 +1,61 @@
+import config from "../../config";
+import IPlayer from "../../interfaces/Player";
+import "./Answer.css";
+
+interface IAnswerProps {
+  correctChoice: string;
+  players: IPlayer[];
+  outOfChunks?: boolean;
+}
+function Answer({ correctChoice, players, outOfChunks }: IAnswerProps) {
+  return (
+    <div className="answer-section">
+      <h2>Leaderboard</h2>
+      {outOfChunks ? (
+        <table>
+          <tr>
+            <th>#</th>
+            <th>Username</th>
+            <th>Score</th>
+          </tr>
+          {players
```

Comment:
We are doing the players.sort() 2 times when displaying. Can we abstract the sorting to another function instead, like getSortedPlayers()?